src/streams.h

```diff
@@ -463,9 +473,15 @@ class CBufferedFile
     uint64_t nReadPos;    // how many bytes have been read from this
     uint64_t nReadLimit;  // up to which position we're allowed to read
     uint64_t nRewind;     // how many bytes we guarantee to rewind
-    std::vector<char> vchBuf; // the buffer
+    typedef boost::intrusive::list<CFileChunk> CChunkList;
+
+    CChunkList chunkList;
+    CChunkList::iterator curChunk;
 
+    //std::vector<char> vchBuf; // the buffer
+    //enum { CHUNK_SIZE = 200000 };  // BU how much additional to allocate if forced to resize
 protected:
+#if 0
```

Comment:
I see that you often comment out code or ifdef it out. Is there a reason ? All history is kept in git, so, as long as PR are kept on point, it is easy to find what was the previous version of the code.